### PR TITLE
Add small example script

### PR DESCRIPTION
**Pull request by Shutgun**
Add a small example script, mainly based on #32.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -137,6 +137,10 @@ asyncio.run(status_codes())
 
 This will give you a [StatusCodes object](https://github.com/ssllabs/ssllabs-scan/blob/master/ssllabs-api-docs-v3.md#statuscodes) as dataclass.
 
+### Example to print the grade of multiple servers
+
+If you are just interested in the grade of servers, you can take [this example](https://github.com/devolo/ssllabs/blob/master/example.py) as s stating point. Just exchange the list of hosts you want to query.
```
Comment (2Fake) [inline README.md:142]:
```suggestion
If you are just interested in the grade of servers, you can take [this example](https://github.com/devolo/ssllabs/blob/master/example.py) as s starting point. Just exchange the list of hosts you want to query.
```

**Review by 2Fake — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -137,6 +137,10 @@ asyncio.run(status_codes())
 
 This will give you a [StatusCodes object](https://github.com/ssllabs/ssllabs-scan/blob/master/ssllabs-api-docs-v3.md#statuscodes) as dataclass.
 
+### Example to print the grade of multiple servers
+
+If you are just interested in the grade of servers, you can take [this example](https://github.com/devolo/ssllabs/blob/master/example.py) as s starting point. Just exchange the list of hosts you want to query.
```
Comment (2Fake) [inline README.md:142]:
```suggestion
If you are just interested in the grade of servers, you can take [this example](https://github.com/devolo/ssllabs/blob/master/example.py) as a starting point. Just exchange the list of hosts you want to query.
```

**Review by 2Fake — CHANGES_REQUESTED** [review]

**Review by 2Fake — APPROVED** [review]

**Comment by malavolti** [comment]
Version in use: 1.2.1

Code affected:

```python
async def analyze(hosts: List[str]) -> List[HostData]:
    """Analyze servers."""
    ssllabs = Ssllabs()
    if not await ssllabs.availability():
        raise
    return await asyncio.gather(*[ssllabs.analyze(host=host,from_cache=True, max_age=1) for host in hosts])
```

GET generated: 
`2024-02-29 11:12:21,204: HTTP Request: GET https://api.ssllabs.com/api/v3/analyze?host=filesender.garr.it&startNew=on&fromCache=off&publish=off&ignoreMismatch=off&maxAge= "HTTP/1.1 200 "`

Seems that "**from_cache**"  parameter doesn't work.

Can you help me, please?

**Comment by Shutgun** [comment]
Interesting ... I'm not seeing that here. Are you sure with the installed version or is maybe a virtual environment in your way?

`2024-02-29 11:38:37,534 httpx INFO: HTTP Request: GET https://api.ssllabs.com/api/v3/analyze?host=devolo.de&startNew=off&fromCache=on&publish=off&ignoreMismatch=off&maxAge=1 "HTTP/1.1 200`

**Comment by malavolti** [comment]
I'm using a VirtualEnvironment to test all.

My VirtualEnv has pip 23.3.1 with:

ssllabs                       1.2.1

**Comment by Shutgun** [comment]
Could you maybe try the following in your script?

```python
from ssllabs import __version__

print(__version__)
```

**Comment by malavolti** [comment]
```
$ python3 ssllabs-version.py

1.2.1
```

**Comment by Shutgun** [comment]
I don't know what going on... If I execute this:

```python
"""Get grade of multiple servers."""
import asyncio
import logging
from typing import List

from ssllabs import Ssllabs, __version__
from ssllabs.data.host import HostData

HOSTS = [
    "devolo.com",
    "www.devolo.com",
]


async def analyze(hosts: List[str]) -> List[HostData]:
    """Analyze servers."""
    ssllabs = Ssllabs()
    if not await ssllabs.availability():
       raise
    return await asyncio.gather(*[ssllabs.analyze(host=host, from_cache=True, max_age=1) for host in hosts])


if __name__ == "__main__":
    print(__version__)
    logging.basicConfig(level=logging.INFO, format="%(asctime)s %(name)s %(levelname)s: %(message)s")
    results = asyncio.run(analyze(HOSTS))
    for result in results:
        for endpoint in result.endpoints:
            if endpoint.grade:
                logging.info(
                    "Grade of %s (%s): %s",
                    result.host,
                    endpoint.ipAddress,
                    endpoint.grade,
                )
            else:
                logging.error(endpoint.statusMessage)
```
I get:
```
1.2.1
2024-02-29 13:54:44,984 httpx INFO: HTTP Request: GET https://api.ssllabs.com/api/v3/info "HTTP/1.1 200 "
2024-02-29 13:54:44,985 ssllabs.Ssllabs INFO: SSL Labs servers are up an running.
2024-02-29 13:54:44,985 ssllabs.Ssllabs INFO: Analyzing devolo.com
2024-02-29 13:54:45,532 httpx INFO: HTTP Request: GET https://api.ssllabs.com/api/v3/info "HTTP/1.1 200 "
2024-02-29 13:54:46,078 httpx INFO: HTTP Request: GET https://api.ssllabs.com/api/v3/analyze?host=devolo.com&startNew=off&fromCache=on&publish=off&ignoreMismatch=off&maxAge=1 "HTTP/1.1 200 "
2024-02-29 13:54:46,080 ssllabs.Ssllabs INFO: Analyzing www.devolo.com
2024-02-29 13:54:46,608 httpx INFO: HTTP Request: GET https://api.ssllabs.com/api/v3/info "HTTP/1.1 200 "
2024-02-29 13:54:47,141 httpx INFO: HTTP Request: GET https://api.ssllabs.com/api/v3/analyze?host=www.devolo.com&startNew=off&fromCache=on&publish=off&ignoreMismatch=off&maxAge=1 "HTTP/1.1 200 "
2024-02-29 13:54:47,143 root INFO: Grade of devolo.com (195.201.179.93): B
2024-02-29 13:54:47,143 root INFO: Grade of devolo.com (159.69.77.193): B
2024-02-29 13:54:47,143 root INFO: Grade of devolo.com (159.69.96.138): B
2024-02-29 13:54:47,143 root INFO: Grade of www.devolo.com (159.69.77.193): B
2024-02-29 13:54:47,143 root INFO: Grade of www.devolo.com (195.201.179.93): B
2024-02-29 13:54:47,143 root INFO: Grade of www.devolo.com (159.69.96.138): B
```

**Comment by 2Fake** [comment]
It would be better to open up an issue on that please.

________________________________

**Comment by 2Fake** [comment]
I just recreated your request with your data to ensure that there isn't something strange with your URL provided. I get the expected result with the result provided from cache. 
Could you test it with provided devcontainer to be sure that your environment is the same as ours?

`2024-02-29 13:19:53,339 httpx INFO: HTTP Request: GET https://api.ssllabs.com/api/v3/analyze?host=filesender.garr.it&startNew=off&fromCache=on&publish=off&ignoreMismatch=off&maxAge=1 "HTTP/1.1 200 "`

**Comment by malavolti** [comment]
I'll try, thank you all. :)

**Pull request merged**